Tests/CategoriesControllerTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using WebNetSample.Business.Abstract;
+using WebNetSample.Entity.Concrete;
+using WebNetSample.Tests.Configurations;
+using WebNetSample.WebNetMVC.Controllers;
+
+namespace WebNetSample.Tests;
+
+public class CategoriesControllerTest
+{
+    [Theory, AutoMoqData]
+    public void Index_Should_Return_As_Expected(List<Category> expected)
+    {
+        // Arrange
+        var productServiceMock = new Mock<ICategoryService>();
+
+        var sut = new CategoriesController(productServiceMock.Object);
+        productServiceMock.Setup(c => c.GetListAsync().Result).Returns(expected);
```

Comment:
Let's give lambda variable more clear name instead of `c`, other person will not understand it from the first view